Dockerfile-refresher

```diff
@@ -38,6 +38,8 @@ WORKDIR /
 COPY --from=builder /etc/nsswitch.conf.build /etc/nsswitch.conf
 COPY --from=builder /usr/share/zoneinfo /usr/share/zoneinfo
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+RUN mkdir /licenses
+COPY --from=builder /workspace/LICENSE.md /licenses
```

Comment:
I wonder if having no subdirectories under `/licenses` will pass the preflight tool check.

`/licenses/imps` instead of `/licenses` should work.

Reply:
looks like just file is enough per error below. yeah can add `/imps/` dir if needed if `preflight` check fails.
```
        "failed": [
            {
                "name": "HasLicense",
                "elapsed_time": 0,
                "description": "Checking if terms and conditions applicable to the software including open source licensing information are present. The license must be at /licenses",
                "help": "Check HasLicense encountered an error. Please review the preflight.log file for more information.",
                "suggestion": "Create a directory named /licenses and include all relevant licensing and/or terms and conditions as text file(s) in that directory.",
                "knowledgebase_url": "https://access.redhat.com/documentation/en-us/red_hat_software_certification/8.45/html/red_hat_openshift_software_certification_policy_guide/assembly-requirements-for-container-images_openshift-sw-cert-policy-introduction",
                "check_url": "https://access.redhat.com/documentation/en-us/red_hat_software_certification/8.45/html/red_hat_openshift_software_certification_policy_guide/assembly-requirements-for-container-images_openshift-sw-cert-policy-introduction"
            }
        ], 
```
see `as text file(s) in that directory` above ^^^